pwiz_tools/Skyline/Model/Results/ChromHeaderInfo.cs

```diff
@@ -174,7 +174,8 @@ public enum FlagValues
             raw_chromatograms = 0x80,
             ion_mobility_type_bitmask = 0x700, // 3 bits for ion mobility type none, drift, inverse_mobility, spares
             dda_acquisition_method = 0x800,
-            extracted_qc_trace = 0x1000
+            extracted_qc_trace = 0x1000,
+            optimization_scan_ids = 0x2000

```

Comment:
I think this flag is either misnamed or unnecessary.

You would not need to separately say whether there are optimization scan id's-- there would be scan ids if "has_ms1_scan_ids" or "has_frag_scan_ids" were set. (I am not sure why there are separate flags for MS1 and MS2-- maybe there really is a mass spectrometer out there which collects MS1 spectra but uses a triple quad for MS2, or vice-versa).

Maybe this flag should be named "optimization" without the "scan_ids" part.